src/commands/convert/prisma2hdf.ts

```diff
@@ -25,7 +25,10 @@ export default class Prisma2HDF extends Command {
     )
     const results = converter.toHdf()
 
-    fs.mkdirSync(flags.output)
+    if (!fs.existsSync(flags.output)) {
```

Comment:
is this correct behavior?  there's a potential for data overwrite to occur if we don't let it crash at this point.  it's also different behavior from the rest of the folder generating cli commands.  if you wanna make it a nicer experience for users, wrap the fs call with a try catch so it can spit a nice error out explaining why the crash happened.

Reply:
I thought this was the same behavior as the prowler2hdf command? It was directly copy/pasted from there, except the file name was changed a bit to reflect the specifics of the PrismaMapper.

Reply:
sounds good then.  might have been an implementation change i wasn't aware about.